### PR TITLE
feat: reactions modal

### PR DESCRIPTION
**Pull request by verbiricha**

### CODE REVIEW

**Comment by v0l** [comment]
Filter duplicate reactions 
image

**Comment by verbiricha** [comment]
@v0l seems that there is a bunch of duplicate reactions, deduped them by pubkey anyway.

**Review by v0l — COMMENTED** [review]
We should also fix loading reposts in timeline view, can be done in this PR

File: src/Element/Reactions.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+const Reactions = ({
+  show,
+  setShow,
+  reactions,
+  reposts,
+  zaps,
+}: ReactionsProps) => {
+  const onClose = () => setShow(false);
+  const likes = useMemo(() => {
+    const positive = reactions.filter((r) => r.content !== "-");
```
[the remark this answers is not in the document]
Comment (verbiricha) [reply src/Element/Reactions.tsx]: refactored to use that and dedupe reactions in footer, this components now just shows them sorted

**Review by v0l — APPROVED** [review]

**Pull request merged**